src/components/Asset/AssetActions/Download/index.tsx

```diff
@@ -338,16 +338,8 @@ export default function Download({
                 />
                 <TermsAndConditionsCheckbox
                   {...content.form.assetTermsAndConditions}
-                  options={[
-                    asset?.metadata?.license.includes('http')
-                      ? 'a custom'
-                      : asset?.metadata?.license
-                  ]}
-                  actions={
-                    asset?.metadata?.license.includes('http')
-                      ? [asset?.metadata?.license]
-                      : undefined
-                  }
+                  options={[asset?.metadata?.license]}
+                  actions={[asset?.metadata?.license]}
```

Comment:
suggestion(blocking): it looks like the previous schema of options and actions does no longer apply to the `TermsAndConditionsCheckbox` component.
It would make sense updating the attributes so it is easier to understand what certain variables are used for.